admin/admin-filters-term.php

```diff
@@ -478,7 +478,10 @@ public function term_lang_choice() {
 					$args = array_merge( $args, array( 'link' => 'edit' ) );
 				}
 
-				if ( $tag_cloud = wp_tag_cloud( $args ) ) {
+				/** @var string */
```

Comment:
```suggestion
				/** @var string|void */
```
Seems more accurate to me.

Reply:
It can't be `void` because of `'echo' => false`.

Reply:
It also returns `void` in case of failure according to the [documentation](https://developer.wordpress.org/reference/functions/wp_tag_cloud/#return).

Reply:
AH !